### PR TITLE
Small change to swap style system in place for credential list

### PR DESCRIPTION
**Pull request by jleach**
# Summary of Changes

Used rebass to style crenendial list so it can follow the theme.

# Related Issues

n/a

# Pull Request Checklist

This is just a reminder about the most common mistakes. Please make sure that you tick all _appropriate_ boxes. But please read our [contribution guide](../CONTRIBUTING.md) at least once; it will save you a few review cycles!

If an item doesn't apply to your pull request, **check it anyway** to make it apparent that there's nothing to do.

- [x] All commits contain a DCO `Signed-off-by` line (we use the [DCO GitHub app](https://github.com/apps/dco) to enforce this).
- [x] Updated LICENSE-3RD-PARTY.md for any added dependencies or vendored components.
- [x] Run prettier: `npm run style-format`
- [x] Updated **documentation** for changed code and new or modified features.

If you have _any_ questions to _any_ of the points above, just **submit and ask**! This checklist is here to _help_ you, not to deter you from contributing!

_PR template adapted from the Python attrs project._

### CODE REVIEW

**Review by amanji — DISMISSED** [review]

File: App/screens/ListCredentials.tsx [hunk 1]
```diff
@@ -1,25 +1,50 @@
...
+
+const MessageText = styled.View`
+  text-align: 'center';
+  margin: 100px;
+`
 
 const ListCredentials: React.FC = () => {
   const { credentials } = useCredentials()
   const { t } = useTranslation()
+  const navigation = useNavigation<StackNavigationProp<any, any>>()
```
Comment (amanji) [inline App/screens/ListCredentials.tsx]: Should be typed? Or is it meant to be generic?

Comment (jleach) [reply App/screens/ListCredentials.tsx]: It is meant to be a work around to a [navigation issue](https://stackoverflow.com/questions/68667766/react-native-typescript-string-is-not-assignable-to-parameter-of-type-never). Without it `navigation.navigate()` coughs up a TS issue 👇🏻 

Screen Shot 2021-11-10 at 3 11 49 PM

Comment (jleach) [reply App/screens/ListCredentials.tsx]: I've looked into this to see if I can put in proper params like <string, any> but it won't take string. Seems to work with `ParamListBase` but I don't know if that is correct or not.

Comment (jleach) [reply App/screens/ListCredentials.tsx]: @amanji Digging into this a bit. Looks fixable but getting a dependency cycle that may relate to other nav warnings I see. Will convert to draft and try to fix.

Comment (jleach) [reply App/screens/ListCredentials.tsx]: Fixed.

**Review by amanji — APPROVED** [review]

**Pull request merged**